app/src/main/java/com/github/se/wanderpals/model/viewmodel/CreateNotificationViewModel.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.se.wanderpals.model.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.github.se.wanderpals.model.data.TripNotification
+import com.github.se.wanderpals.model.repository.TripsRepository
+import kotlinx.coroutines.launch
+
+/**
+ * ViewModel for creating notifications. This ViewModel is responsible for adding a notification to
+ * a trip.
+ */
+open class CreateNotificationViewModel(tripsRepository: TripsRepository) : ViewModel() {
+  private val _tripsRepository = tripsRepository
+

```

Comment:
It could be nice to leave a comment to describe what the function does to improve code readability.

Reply:
Yes, I added it! Please have a check :D